pkg/apis/serving/v1alpha2/inferenceservice_validation.go

```diff
@@ -34,7 +34,7 @@ const (
 )
 
 var (
-	SupportedStorageURIPrefixList = []string{"gs://", "s3://", "pvc://", "file://"}
+	SupportedStorageURIPrefixList = []string{"gs://", "s3://", "pvc://"}
```

Comment:
why remove "file://"? for `Custom` type, user can include both model and inference app  in local image

Reply:
Because it does not work so I suggest to either fix it or remove it. Considering it's broken for a long time and no one reported it so I think it's not used by anyone. We can consider fixing it too if it's still useful.